recipes-ni/ni-base-system-image-tests/files/fs_permissions_known.py

```diff
@@ -27,13 +27,161 @@ def known_permissions_tree():
                 'bzImage': system_link(f'bzImage-{bash("$(uname -r)")[0]}'),
             }
         },
+        'etc': {
+            'fstab': permissions(0o0644, 'lvuser', 'ni', FT_REG)
+        },
+        'home': {
+            '.': system_dir,
+            '*': system_dir,
+            'lvuser': permissions(0o2775, 'lvuser', 'ni', FT_DIR),
+            'root': system_link('/home/admin'),
+            'webserv': permissions(0o2755, 'webserv', 'ni', FT_DIR)
+        },
         'lib': {
             '.': system_dir,
             'modules': {
                 '.': system_dir,
                 '$(uname -r)': {
                     '.': system_dir,
                     '**': system_hier,
+                    'build': {
+                        'arch': {
+                            'x86': {
+                                'tools': {
+                                    'relocs': system_file_exec
+                                }
+                            }
+                        },
+                        'include': {
+                            'dt-bindings': {
+                                'clock': {
+                                    'qcom,dispcc-sm8150.h': system_link('qcom,dispcc-sm8250.h')
+                                },
+                                'input': {
+                                    'linux-event-codes.h':
+                                        system_link('../../uapi/linux/input-event-codes.h')
+                                }
+                            }
+                        },
+                        'scripts': {
+                            '**/*.sh': system_file_exec,
+                            '**/*.pl': system_file_exec,
+                            'asn1_compiler': system_file_exec,
+                            'atomic': {
+                                'atomics.tbl': system_file_exec,
+                                'fallbacks': {
+                                    '*': system_file_exec
+                                }
+                            },
+                            'basic': {
+                                'fixdep': system_file_exec
+                            },
+                            'bloat-o-meter': system_file_exec,
+                            'bpf_doc.py': system_file_exec,
```

Comment:
Could we cover this and other .py entries with a **/*.py rule as done above with .sh and .pl?